modules/dnn/src/net_impl.cpp

```diff
@@ -1830,15 +1830,266 @@ string Net::Impl::dump(bool forceAllocation) const
     return out.str();
 }
 
+static void dumpShapeToString(std::ostringstream &out, const Mat &m, const int num_indent_spaces = 4) {
```

Comment:
Not only shape. Maybe `dumpTensorToString`?

Reply:
You are right. Renamed to `dumpTensorToString`.